app/src/main/res/values/strings.xml

```diff
@@ -72,4 +65,22 @@
     <string name="enter_server">Please enter your server URL</string>
     <string name="auth_login">Your server allows these Authentication Options:</string>
     <string name="another_server">Another server</string>
-</resources>
+    <string name="message_sent">Message Sent!</string>
```

Comment:
`Message sent!`

Note that it's better to only display messages when something has failed.  The user will know their message has been sent because it will appear in the chat. :)